client/src/components/gamedashboard/global/containers/ContainerDefeat.vue

```diff
@@ -1,23 +1,34 @@
 <template>
   <div class="container-defeat">
-    <h1 class="animated pulse slower infinite">Game Over</h1>
-    <div class="text">
-      <p>
-        System Health has reached zero. However, your efforts will inspire a new
-        generation to continue your legacy on Mars.
-      </p>
-      <p>Thank you for playing.</p>
+    <h1 class="animated pulse slower infinite gameOver-text">Game Over</h1>
+    <div class="transmission-info">
+      <h3>Replaying Logs before death...</h3>
+      
```

Comment:
We probably want to keep the `System Health has reached zero...` text (or something like it) here to keep the narrative in addition to the pile of log events.

Reply:
The last log that comes in says that system health has reached zero. We can re-iterate it if you want, but it is there.

Reply:
There needs to be a narrative goodbye, which I don't expect to exist in the mars log. Don't worry too much about this, I can add it in later.